Xamarin.Forms.Core/Device.cs

```diff
@@ -232,6 +232,12 @@ public static void OpenUri(Uri uri)
 
 		public static void StartTimer(TimeSpan interval, Func<bool> callback)
 		{
+			if (interval.TotalMilliseconds == 0)
+			{
+				while (callback())
+					;
+				return;
```

Comment:
nothing else will ever get a chance to get executed. is this what we want ? Shouldn't we have a minimal (internal) value of 15ms ?

Reply:
what if the user wants 0ms, that's actual the use case